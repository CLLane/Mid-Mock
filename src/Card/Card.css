
img {
  height: 150px;
  width: 150px;
  justify-content: start;
}

