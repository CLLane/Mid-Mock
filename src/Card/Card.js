import React from 'react';
import './Card.css'
const Card = ({ data }) => {
  return (
    <article className='purchase-container'>
      <img src={data.img} alt=''/>
      <h1>{ data.name }</h1>
      <p>{ data.description }</p>
      <p>${data.price}</p>
    </article>
  ) 
}

export default Card;