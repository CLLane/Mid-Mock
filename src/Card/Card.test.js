import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Card from './Card';

describe('Card', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<Card 
      data=''
    />)
    expect(wrapper).toMatchSnapshot()
  })

  
})