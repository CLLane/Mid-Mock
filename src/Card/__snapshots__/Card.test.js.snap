// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot 1`] = `
<article
  className="purchase-container"
>
  <img
    alt=""
  />
  <h1 />
  <p />
  <p>
    $
  </p>
</article>
`;
